socorro/unittest/external/postgresql/test_base.py

```diff
@@ -135,11 +139,9 @@ def test_utc(self):
         # Verify that we've got 'timezone=utc' set
         sql = 'SHOW TIMEZONE'
         results = base.query(sql)
-        ok_(
-            'UTC' in results[0],
-            """Please set PostgreSQL to use the UTC timezone.
-               Documentation on how to do this is included in
-               the INSTALL instructions."""
+        assert 'UTC' in results[0], (
+            'Please set PostgreSQL to use the UTC timezone. Documentation on how to do this is '
+            'included in the installation instructions.'
```

Comment:
I kept the assertion comment here. I find it pretty curious, though. What happens if postgresql isn't using UTC? Do other tests fail?